src/plugins/LADTable/components/LadRow.vue

```diff
@@ -48,6 +48,8 @@
 </template>
 
 <script>
+import throttle from '../../../utils/throttle';
+const waitForMs = 1000;
```

Comment:
Two kinda architecture questions:

- should we make this 1000ms a default for throttle?
- should we keep `raf`?